### ISSUE TITLE
Warning: Failed prop type: The prop `detailPanel` is marked as required

### ISSUE THREAD

**Issue by jvlobo** [mbrn/material-table#144]
I don't know why I'm getting this "Warning" that shows as an error in the console.

This is my definition of the component:

```
<MaterialTable
  columns={[
    { title: 'Title', field: 'title' },
    { title: 'Description', field: 'description' },
  ]}
  data={videos}
  title="Videos"
/>
```

<img width="1105" alt="screenshot 2019-01-09 at 21 24 23" src="https://user-images.githubusercontent.com/5671420/50896569-16df0100-1455-11e9-937a-f3f9a862b59e.png">

Is this normal?

Thanks.

**Comment by mbrn** [comment]
Hi @jvlobo ,

This is not normal. It was my mistake on previous version. I fixed it on new version (1.12.0). Can you check your version. I published it today.

**Comment by jvlobo** [comment]
Right! I had v1.10, just updated and working great 👍 
Thanks!
